prometheus/prometheus.py

```diff
@@ -162,7 +162,7 @@ def collect(self):
         peers = self.rpc.listpeers()['peers']
         for p in peers:
             for c in p['channels']:
-                labels = [p['id'], c['channel_id']]
+                labels = [p['id'], c.get('short_channel_id', c.get('channel_id'))]
```

Comment:
Doesn't this now mismatch label keys with label values? We probably need to either add the long ID to the gauge and counter declarations, or drop it here (I think `short_channel_id` is sufficient and more memorable?).

Reply:
The label was `scid`, so it mismatched before (I just kept the long id in case the channel isn't yet confirmed and `short_channel_id` isn't a field of `listpeers` yet)